projects/showcase/src/styles.scss

```diff
@@ -16,6 +18,28 @@ body {
 .mat-badge-content {
   background: none;
 }
+
 .mat-tooltip {
   font-size: 14px;
 }
+
+.range-selector {
+  .mat-form-field {
+    top: 5px !important;
+  }
+
+  .mat-form-field-infix {
+    display: flex !important;
+    height: 40px;
+  }
+
+  input.mat-input-element {
+    position: relative !important;
+    bottom: 0px !important;
+  }
+
+  .mat-datepicker-toggle {
+    position: relative !important;
+    bottom: 20px !important;
+  }
```

Comment:
custom styles are needed because if I remove custom styles it's not look in the proper format.
I am attaching a screenshot for you to look over.
![Screenshot from 2023-01-04 17-59-52](https://user-images.githubusercontent.com/117890382/210555630-8fe3ca4e-9588-48ef-aabe-dc54865eb08a.png)

Reply:
If I add custom CSS it looks better and in a proper format.
I am attaching a screenshot for you to take a look at.
![image](https://user-images.githubusercontent.com/117890382/210557826-dc931f00-b602-4a1a-bec0-a4f13f25f5a0.png)

Reply:
Thanks, I will take a look

Reply:
Okay, I am changing matLconSuffix to matSuffix and Also removing &nbsp.